doc/how_to/migrate/from_streamlit.md

```diff
@@ -0,0 +1,950 @@
+)
+
+pn.Row(col1, col2).servable()
+```
+
+Panels `Column` and `Row` are *list like* objects. So you can use familiar methods like `.append`,
+`.pop` and `[]` indexing when you work with them. For the details check out the
+[`Column` Guide](https://panel.holoviz.org/reference/layouts/Column.html) and
+the [`Row` Guide](https://panel.holoviz.org/reference/layouts/Row.html).
+
+The app looks like
+
+![Panel Layout Example](https://user-images.githubusercontent.com/42288570/243362603-45ba78a4-d67b-43bc-b3c2-386105fe6ed8.png)
+
+### Layout Migration Steps
+
+You should
+
+- replace your Streamlit `st.some_layout` function with the corresponding Panel
+`pn.SomeLayout` class.
+
+You can identify the relevant layout to migrate to in the
+[Layouts Section](https://panel.holoviz.org/reference/index.html#layouts) of the
+[Component Gallery](https://panel.holoviz.org/reference/index.html).
+
+### Template Migration Steps
+
+When migrating you first have to choose which template to use
+
+- None (default)
+- A built in like *vanilla*, *bootstrap*, *material* or *fast*. See the
+[Templates Section](../../reference/index#templates) of the
+[Components Guide](../../reference/index).
+- A custom template
+
+Then you have to configure it.
+
+#### Panel Template Example
+
+Here is an example with the [`FastListTemplate`](https://panel.holoviz.org/reference/templates/FastListTemplate.html) for illustration.
+
+```python
+import panel as pn
+from datetime import datetime
+from asyncio import sleep
+import panel as pn
+from asyncio import sleep
+
+pn.extension(sizing_mode="stretch_width", template="fast", theme="dark")
+
+pn.Column(
+    "# 📖 Info",
+    """This app is an example of a built in template with a
+*sidebar*, *header* and *main* area.
+
+We have
+
+- set the *header* background, site and title parameters
+- set the default *theme* to `dark`
+
+The app streams the current date and time using an *async generator function*.
+""",
+).servable(target="sidebar")
+
+
+async def stream():
+    for i in range(0, 100):
+        await sleep(0.25)
+        yield datetime.now()
+
+
+pn.Column("The current date and time:", *(stream for i in range(5))).servable(
+    target="main"
+)
+
+pn.state.template.param.update(
+    site="Panel",
+    title="Template Example",
+    header_background="#E91E63",
+    accent_base_color="#E91E63",
+)
+```
+
+The app looks like
+
+![Panel Template Example](https://user-images.githubusercontent.com/42288570/243438919-edce17a0-48b6-451d-9be3-d86eff5cc166.gif)
+
+## Show Activity
+
+Panel supports two ways of indicating activity
+
+- Indicators. See the [Indicators Section](https://panel.holoviz.org/reference/index.html#indicators)
+of the [Component Gallery](https://panel.holoviz.org/reference/index.html).
+- `disabled`/ `loading` parameters on Panel components
+
+We will show you how to migrate your Streamlit activity indicators to Panel in the
+[Interactivity Section](#interactivity) just below.
```

Comment:
Maybe this can come after Interactivity and show an example of using `loading_indicator`

`pn.panel(pn.bind(...), loading_indicator=True)`

Reply:
I thought about this. But could not see how I could do this in a way that made sense to me. Feel free to contribute an improvement.